packages/docs/public/_redirects

```diff
@@ -47,7 +47,7 @@
 /qwikcity/static-site-generation/overview/ /docs/guides/static-site-generation/ 308
 /qwikcity/static-site-generation/static-site-config/ /docs/guides/static-site-generation/ 308
 /qwikcity/static-site-generation/dynamic-routes/ /docs/guides/static-site-generation/ 308
-/qwikcity/advanced/prefetching/ /docs/advanced/speculative-module-fetching/
+/qwikcity/advanced/prefetching/ /docs/advanced/modules-prefetching/
```

Comment:
Changed the redirect to `/modules-prefetching`, since the first paragraph in `/modules-prefetching` already redirects to `/speculative-module-fetching`.